From: Chris Lamb <lamby@debian.org>
Date: Fri, 5 Apr 2019 15:48:23 +0200
Subject: Mark non-determinstic test_minimize_dfa test as XFAIL

---
 tests/test_automata.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_automata.py b/tests/test_automata.py
index daab96c..7031dc3 100644
--- a/tests/test_automata.py
+++ b/tests/test_automata.py
@@ -327,6 +327,7 @@ def test_regular():
     assert not ex.accept("bc")
 
 
+@pytest.mark.xfail(strict=False)
 def test_minimize_dfa():
     # Example from www.cs.odu.edu/~toida/nerzic/390teched/regular/fa/min-fa.html
 
